### PR TITLE
New version: dorssel.usbipd-win version 2.0.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41496)

### CODE REVIEW

**Comment by dorssel** [comment]
Where does this PR come from?
See #41495, which does exactly the same. BTW, the 'dorssel' in 'dorssel.usbipd-win' is me.
This seems to be a duplicate.

File: manifests/d/dorssel/usbipd-win/2.0.0/dorssel.usbipd-win.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+PackageLocale: en-US
+Publisher: Frans van Dorsselaer
+PublisherUrl: https://github.com/dorssel
+PublisherSupportUrl: https://github.com/dorssel/usbipd-win/issues
+# PrivacyUrl: 
+# Author: 
+PackageName: usbipd-win
+PackageUrl: https://github.com/dorssel/usbipd-win
+License: GNU General Public License, version 2
+LicenseUrl: https://github.com/dorssel/usbipd-win/blob/v1.3.0/COPYING.md
```
Comment (dorssel) [inline manifests/d/dorssel/usbipd-win/2.0.0/dorssel.usbipd-win.locale.en-US.yaml:15]: The version is not updated here. 1.3.0 -> 2.0.0

**Comment by vedantmgoyal2009** [comment]
> Where does this PR come from? See #41495, which does exactly the same. BTW, the 'dorssel' in 'dorssel.usbipd-win' is me. This seems to be a duplicate.

There must be a difference of a few minutes and that's why automation can't detect that there is already an existing PR.

> Is this automation? Does this mean I will never have to inject new versions myself again?

Yeah! This PR comes from automation at https://github.com/vedantmgoyal2009/winget-pkgs-automation.

**Review by OfficialEsco — APPROVED** [review]
Approving this one since it upgrades the ManifestVersion to 1.1.0

File: manifests/d/dorssel/usbipd-win/2.0.0/dorssel.usbipd-win.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+PackageLocale: en-US
+Publisher: Frans van Dorsselaer
+PublisherUrl: https://github.com/dorssel
+PublisherSupportUrl: https://github.com/dorssel/usbipd-win/issues
+# PrivacyUrl: 
+# Author: 
+PackageName: usbipd-win
+PackageUrl: https://github.com/dorssel/usbipd-win
+License: GNU General Public License, version 2
+LicenseUrl: https://raw.githubusercontent.com/dorssel/usbipd-win/master/COPYING.md
```
Comment (dorssel) [inline manifests/d/dorssel/usbipd-win/2.0.0/dorssel.usbipd-win.locale.en-US.yaml:15]: I'll accept this change. Originally this was a URL that pointed to the license *for this specific version of the software*. Given that this software is GPL2.0-only, which is sticky and thus permanent, it doesn't really matter. But please consider allowing a version-dependent LicenseUrl. For other software, the license in the master branch may not represent the license for this specific version of the installer.

**Review by dorssel — APPROVED** [review]

File: manifests/d/dorssel/usbipd-win/2.0.0/dorssel.usbipd-win.locale.en-US.yaml (hunk 2 above)
Comment (vedantmgoyal2009) [reply manifests/d/dorssel/usbipd-win/2.0.0/dorssel.usbipd-win.locale.en-US.yaml:15]: Okay

**Pull request merged**